### ISSUE TITLE
New search UI (1.52): gray out or hide filtered commits

### ISSUE THREAD

**Issue by Otiel** [ForkIssues/TrackerWin#879]
I like the new search UI as it allows us to quickly see the found commits in a separate list. 👍 

But what has been lost compared to the old search is that commits from filtered out branches where not taken into account when searching. Now all branches are searched. It's disturbing to: a) see commits that are not from the filtered branches, and b) when clicking on a result, having the commit shown but nothing selected in the commits list.

I'm not sure what would be the best solution:
* Graying out commits that are not from the filtered branches, or,
* Don't show commits that are not from the filtered branches.

Anyway, I think one of those solution would be better than the actual behavior.

**Comment by ghost** [comment]
I agree with this. The new search UI makes it much easier to navigate through the search results, but that doesn’t help when I have one branch or two filtered, and the results show the commits of all branches. 

I think there should be an option to search only within the filtered branch(es). Also, some indicator on the search result that would display what branch the search result is in would greatly help.

Appreciate everything you are doing with this awesome product!!
